katello/hooks/boot/30-upgrade.rb

```diff
@@ -1,12 +0,0 @@
-app_option(
```

Comment:
Rather than removing, should we keep it and error out with a message that it's no longer needed for a release or 2?